### DISCUSSION TITLE
Regarding Third Party library versions

### DISCUSSION THREAD

**Discussion by vishalrkb** [GrapesJS/grapesjs#4156] [Q&A]
Hi 

We are trying get approval for grapesjs for one of our applications in our organization. The problem we are facing in getting approval is third party library versions ex: backbone 1.3.3, it is 2016 version, are there any plans to upgrade these versions to latest?

**Comment by artf** [comment]
If there are no security issues with dependencies there is no point to upgrade them as they are considered features complete from the framework point of view.
